pipelines/short-read-alignment-vector/ShortReadAlignment.options.json

```diff
@@ -0,0 +1,5 @@
+{
+  "read_from_cache": true,
+  "write_to_cache": true,
+  "monitoring_script": "gs://broad-gotc-test-storage/cromwell_monitoring_script.sh"
```

Comment:
FYI - this is a script that monitors mem/disk/cpu usage on google VMs in parallel with the executing task - very helpful for figuring out how much memory/disk a task needs.